PluginAutotask/API/Discover/GetRefreshSchemas.cs

```diff
@@ -1,25 +1,30 @@
 using System.Collections.Generic;
 using Google.Protobuf.Collections;
-using Naveego.Sdk.Plugins;
-using PluginHubspot.API.Factory;
-using PluginHubspot.API.Utility;
-using PluginHubspot.Helper;
+using Aunalytics.Sdk.Plugins;
+using PluginAutotask.API.Factory;
+using PluginAutotask.API.Utility;
+using PluginAutotask.Helper;
 
-namespace PluginHubspot.API.Discover
+namespace PluginAutotask.API.Discover
 {
     public static partial class Discover
     {
-        public static async IAsyncEnumerable<Schema> GetRefreshSchemas(IApiClient apiClient, Settings settings,
-            RepeatedField<Schema> refreshSchemas, int sampleSize = 5)
+        public static async IAsyncEnumerable<Schema> GetRefreshSchemas(IApiClient apiClient, RepeatedField<Schema> refreshSchemas, int sampleSize = 5)
         {
-            foreach (var schema in refreshSchemas)
+            foreach (var refreshSchema in refreshSchemas)
             {
-                var endpoint = EndpointHelper.GetEndpointForSchema(schema);
+                var schema = refreshSchema;
+                // not user defined
+                if (Constants.EntitiesList.Contains(schema.Id))
+                {
+                    schema = await AddPropertiesForEntity(apiClient, schema);
+                    schema = await AddSampleAndCount(apiClient, schema, sampleSize);
 
-                var refreshSchema = await GetSchemaForEndpoint(apiClient, schema, endpoint);
+                    yield return schema;
+                }
 
-                // get sample and count
-                yield return await AddSampleAndCount(apiClient,  refreshSchema, settings, sampleSize, endpoint);
+                // TODO: add support for user defined schemas
```

Comment:
Is this supposed to remain in TODO? Perhaps for phase 2?

Reply:
It is meant to be TODO. It is not in scope but I left hooks for us to write it in later if we were asked to make custom user 
defined schemas.